### DISCUSSION TITLE
Smart Charging (registers)

### DISCUSSION THREAD

**Discussion by Triggerfinger123** [mletenay/home-assistant-goodwe-inverter#274] [Q&A]
Hi,

I am trying to implement the 'Smart Charging Mode' in node-red for an 20 ET inverter.  
Smart Charging Mode Switch on register 47609 works.  
But witch register do I have to use to control the "Peak Limiting Power"?

![image](https://github.com/mletenay/home-assistant-goodwe-inverter/assets/151454502/fcb08323-ec3e-41c3-8fc4-a04df935503b)

**Comment by ccspack** [comment]
Did anyone get address to do this yet?
